server/item/armour.go

```diff
@@ -27,6 +27,9 @@ type (
 		// KnockBackResistance is a number from 0-1 that decides the amount of knock back force that is resisted
 		// upon being attacked. 1 knock back resistance point client-side translates to 10% knock back reduction.
 		KnockBackResistance() float64
+		// EnchantmentValue is the enchantment value of the armour used when selecting pseudo-random enchantments for
+		// enchanting tables.
```

Comment:
What does that imply? Better enchantments when this value is higher? Might want to document that a bit more thoroughly.

Reply:
Improved the documentation here to include that.